packages/ui/src/testHelpers.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
💭 You can add `'**/testHelpers.tsx'` to `FILES_WITH_DEV_DEPENDENCIES` array in the `.eslintrc.js`